### PR TITLE
Ready for review

### PR DESCRIPTION
**Pull request by noomdalv**

### CODE REVIEW

**Review by stratospherique — COMMENTED** [review]
### Pull request under the review by Ahmed Mahfoudh (TSE)

File: index.css [hunk 1]
```diff
@@ -0,0 +1,884 @@
...
+    width: 100%;
+  }
+
+  .sixthRight {
+    display: block;
+    width: 100%;
+  }
+}
+
+@media (max-width: 768px) {
```
Comment (stratospherique) [inline index.css:858]: You should use Bootstrap default classes for responsiveness. :no_entry_sign:

**Review by stratospherique — CHANGES_REQUESTED** [review]
## Hi @noomdalv , Great work. However, some requirements needs to be implemented in order for us to approve your project:

##### Featured story
- In that section, you have some text that sits on top of it. Please remove
image

### footer ( also subscribe and newsletter )

- Your are using responsive layout there. However font size and some of the elements is not responsive.

#### Implementation details

* [ ] add the bootstrap [viewport metatag](https://getbootstrap.com/docs/4.3/getting-started/introduction/#responsive-meta-tag)
* [ ] It uses Bootstrap Css classes for styling ( for example [Grid layout](https://getbootstrap.com/docs/4.3/layout/overview/) and [Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options) )

### General notes:

* [ ] The project has to have a descriptive README.md

**Comment by stratospherique** [comment]
### And that's it. When finished implementing the above, kindly resubmit another code review. 
#### Keep up the good work :muscle: :muscle:

**Comment by noomdalv** [comment]
PROJECT READY

1. Bootstrap metatag implemented

2. Responsiveness added for:
Navbar List
Subscribe section
Newsletter section
Footer

3. Project README updated

File: index.css [hunk 2]
```diff
@@ -0,0 +1,884 @@
...
+  font-size: 30px;
+  width: max-content;
+}
+
+
+@media (max-width: 992px) {
+  #wrapper {
+    width: 100%;
+  }
+
```
Comment (stratospherique) [inline index.css:802]: - Again these media queries are necessary. You supposed to use bootstrap classes.

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great work :clap: :clap: . Unfortunately, you missed the most important goal of this project. **You should be using Bootstrap default classes** for responsive positioning and layout. :disappointed: 

* [ ] You should use Bootstrap Css classes **everywhere** for styling ( for example [Grid layout](https://getbootstrap.com/docs/4.3/layout/overview/) and [Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options) )

* [ ] **You should add linter configuration :no_entry_sign:  It's mandatory. You could check Microverse's [repo](https://github.com/microverseinc/linters-config/tree/master/css), it will walk through the process of the configuration**.

#### Please take care of this important matter :pray: . When finished, submit another code review request

**Comment by noomdalv** [comment]
Stickler added, linter warnings such as (fallback-colors) and selectors (no-descending-specificity) errors not possible to be corrected with fixer enabled so ignore those.

Classes for bootstrap are implemented throught the whole project, most of the positioning was done using bootstrap flex properties, please review project before giving any non true suggestions, thanks

**Review by uche-inyama — COMMENTED** [review]
## Hi there, great job so far :fire: 
Kindly ensure that this project passes the linter checks.

### When you are done, submit another form for code review.

**Comment by noomdalv** [comment]
Stickler errors fixed, project finished.

**Review by stratospherique — APPROVED** [review]
### Great Work :clap: . All requirements are met :100: . Your project is approved :+1: :+1: :+1:  and you can merge to master. Keep up the good work :muscle:

**Pull request merged**